ta-sdk-sample/src/test/java/com/ibm/ta/sdk/sample/test/MigrationArtifactsHelperTest.java

```diff
@@ -0,0 +1,69 @@
+package com.ibm.ta.sdk.sample.test;
+
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+import com.ibm.ta.sdk.core.util.GenericUtil;
+import com.ibm.ta.sdk.core.util.MigrationArtifactsHelper;
+import com.ibm.ta.sdk.spi.plugin.TADataCollector;
+import com.ibm.ta.sdk.spi.plugin.TAException;
+import com.ibm.ta.sdk.spi.util.Util;
+import freemarker.template.TemplateException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MigrationArtifactsHelperTest {
+
+    static final String MIDDLEWARE_NAME = "middleware";
+    static final String COLLECTION_UNIT_NAME = "collection1";
+    static final String ASSESS_UNIT_NAME = "assessmentUnit1";
+
+    @BeforeEach
+    void cleanOutpotDir() throws IOException {
+        Path outputDir = Util.getOutputDir().toPath();
+        if (outputDir.toFile().exists()) {
+            Files.walk(outputDir)
+                    .sorted(Comparator.reverseOrder())
+                    .map(Path::toFile)
+                    .forEach(File::delete);
+        }
+    }
+
+    @Test
+    public void generateMigrationFilesForTargetTest() throws IOException, URISyntaxException, TemplateException, TAException {
+        List<String> argus = new ArrayList<>();
+        argus.add("run");
+        argus.add("test");
+        TADataCollector tadc = new TADataCollector();
+        tadc.runCommand(MIDDLEWARE_NAME, argus);
+        String templateFilePath = "src/main/resources/middleware/templates/targetA";
```

Comment:
Do the default paths need to be FileSeperator instead of "/" ?

Reply:
This path used in Junit, it is used by classloader,  and represented as the classpath,  back slash is fine.